### PR TITLE
SNOW-795117: Bumped up PythonConnector PATCH version from 3.0.2 to 3.0.3

### PR DESCRIPTION
**Pull request by sfc-gh-hchoi**
@noreview - This is an automated process. No review is required

Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
 - v3.0.3(TBD)
 
+- v3.0.3(April 20,2023)
+
+   -N/A
+
+
   - Fixed a bug that prints error in logs for GET command on GCS.
   - Added a parameter that allows users to skip file uploads to stage if file exists on stage and contents of the file match.
   - Fixed a bug when writing a Pandas DataFrame with non-default index in `snowflake.connector.pandas_tool.write_pandas`.
```
Comment (sfc-gh-mheavin) [inline DESCRIPTION.md:20]: Fixed a bug that occurred when ...

**Review by sfc-gh-mheavin — APPROVED** [review]
Minor suggestion

File: DESCRIPTION.md (hunk 1 above)
Comment (sfc-gh-kdama) [reply DESCRIPTION.md:14]: @sfc-gh-mheavin  Please commit the recommendation here in this branch.

Comment (sfc-gh-mheavin) [reply DESCRIPTION.md:14]: @sfc-gh-kdama I don't have the git repo yet. Whoever added the notes can choose to include my suggestion.

Comment (sfc-gh-mkeller) [reply DESCRIPTION.md:14]:
```suggestion
  - Fixed a bug that occurred when writing a Pandas DataFrame with non-default index in `snowflake.connector.pandas_tool.write_pandas`.
```

Comment (sfc-gh-mheavin) [reply DESCRIPTION.md:14]: @sfc-gh-mkeller Can I just commit here directly in git instead of cloning the repo and checking out the branch?

**Review by sfc-gh-sfan — APPROVED** [review]

File: DESCRIPTION.md (hunk 1 above)
Comment (sfc-gh-kdama) [reply DESCRIPTION.md:14]: you can do that to @sfc-gh-mheavin  going forward.

**Review by sfc-gh-aling — APPROVED** [review]

**Review by sfc-gh-kdama — APPROVED** [review]

**Pull request merged**